### ISSUE TITLE
Remove Fairy Tales 2 Redux: A Tale or Two

### ISSUE THREAD

**Issue by dragonrider455** [StratusNetwork/Rotations#161]
The map has 2 sides, Red and Blue.  Under each side, there are 2 types up blocks decorating the siding.  Nether-rack and Nether Brick for red, Stone bricks and cobble stone for blue.  I tried mining them each with a pick-axe in a survival world, nether-rack breaks faster than the other 3, which are the same.  Therefore, the sides are unequal.  While this isn't all that big of an issue,  the map hurts to look at.  Bad trees, ugly wool lanes, and way to much glow-stone end up making it look really bad.  The wool lane is essentially a path at a slight upward angle with one large lump, that doesn't look natural or good.  It also has ALOT of glow-stone in the stream coming out of it, which looks terrible. It also isn't that cool that you basically have to run above the enemy spawn to get into a wool room.

**Comment by Palmidence** [comment]
It's a classic map that's essentially very popular. Rather than remove it, I'd like to explore further options in optimizing the map to play better. We can easily figure out a solution to unequal sides and wool lanes, and retain the best features of the map. I wouldn't just junk it.

**Comment by Matic0B** [comment]
I don't think we should remove it. Like Vice said, it is a classic map that is very popular even though it has some flaws. I'd rather improve the issues.
